clouds/bigquery/modules/doc/processing/ST_DELAUNAYLINES.md

```diff
@@ -39,7 +41,22 @@ SELECT `carto-os`.carto.ST_DELAUNAYLINES(
 -- LINESTRING(-72.514071762468 36.5823995124737, -73.3262122666779 ...
 ```
 
-{{% bannerNote type="note" title="ADDITIONAL EXAMPLES"%}}
+Note that if some points are very close together (about 1 meter) they may be merged and the result may have fewer lines than expected, for example these four points result in two lines:
```

Comment:
If there is more than one example we should write `Examples` in this section for all the docs. But as it is explaining a technical limitation I'm not sure that is needed.